enhancements/sdk-fbc-run-bundle.md

```diff
@@ -0,0 +1,279 @@
+
+- Operator SDK's `run bundle` command should still be able support SQLite index images along with the FBC images until `opm registry add` and `opm index` and other SQLite-related commands are removed in a future OpenShift release.
+
+- The CLI for `run bundle` command should remain exactly the same i.e. no user-facing changes.
+
+### Non-Goals
+
+- Operator SDK’s `run bundle` and `run bundle-upgrade` commands share a lot of common functionality and this EP does not discuss the impact on the run bundle-upgrade command. A separate EP is required for the `run bundle-upgrade` command.
+
+- Migrating from old catalog images to newer catalog versions with FBC support is not a goal of this EP.
+
+- Another non-goal, is that `run bundle` is **NOT** intended to help users build and manage File-Based indexes. It is an opinionated command to help users test simple bundle-based execution scenarios with ephemeral indexes.
+
+## Proposal
+
+### User Stories
+
+#### Story 1
+
+As an Operator SDK user, I want to be able to install an operator bundle with File-Based Catalog support using OLM without having to provide a catalog.
+
+#### Story 2
+
+As an Operator SDK user, I want to be able to install an operator bundle with File-Based Catalog support using OLM when I want to use an existing File-Based Catalog as the index image.
+
+#### Story 3
+
+As an Operator author, I want to use the Operator SDK’s `run bundle` command that has support for both File-Based Catalog and SQLite index images.
+
+#### Story 4
+
+As an Operator SDK user, I want to be able to use the Operator SDK’s `run bundle` command to install the user provided bundle image with an index image that does not have FBC support (i.e. SQLite based images).
+
+### Implementation Details/Notes/Constraints [optional]
+
+The Operator SDK currently supports running an operator bundle using OLM via the run bundle command, which is suited for the SQLite-based index format. This enhancement will address how the Operator SDK will support the new File-Based Catalog format for operator catalogs.  More Information on FBCs and how operator bundles are created using Operator SDK can be found at the links below:
+
+- [File-Based Catalogs](https://olm.operatorframework.io/docs/reference/file-based-catalogs/)
+- [File-Based Catalog Issue](<https://github.com/k8s-operatorhub/community-operators/discussions/505>)
+- [Operator-Bundles](<https://github.com/operator-framework/operator-registry/blob/master/docs/design/operator-bundle.md>)
+
+The declarative config is a struct defined by the Operator Registry that represents the index in a format that is easily transformed to a `YAML/JSON` format. The Operator SDK will invoke Operator Registry APIs to generate various components of a declarative config and eventually insert them into the declarative config object. This object will then be converted to a human-readable form like `YAML/JSON`.
```

Comment:
We could be more specific here:

The declarative config is a struct defined by the Operator Registry that represents a File-Based Catalog in a format that is easily transformed to a YAML/JSON format.

"Index" could also mean a SQLite index, but declarative config objects would only correspond to FBCs.

Reply:
Why do we need to describe what is FBC on this EP?
The goal is to define how run bundle will work and not how FBC works and its motivation
in POV it is not an ep for FBC so all this content is not relevant at all.

Reply:
As I mentioned in another comment, we want to provide context around the FBC to anyone who reads this EP that may not have extensive knowledge about FBCs. Additionally, we are constructing the FBC in the SDK code, so we felt it was important to outline how we are constructing the FBC. This involves detailing some components. A majority of this EP still discusses the changes proposed. We just wanted to provide some additional context.

Reply:
I don't think we are describing the details of FBC here. There's a separate merged enhancement for FBC altogether. This EP for run bundle with FBC support aims to explain the details of how an FBC or declarative config is created during the run bundle process. 

I like the idea of having a a very short description of what FBC is. It makes it easy for folks who are not completely familiar with the actual FBC EP who'd like to review this proposal and understand how FBC impacts SDK's `run bundle` command and the changes we are proposing.